scylla/src/transport/load_balancing/default.rs

```diff
@@ -984,6 +1009,31 @@ mod tests {
                     .group([]) // remote nodes
                     .build(),
             },
+            // Keyspace NTS with RF=3 with enabled DC failover, shuffling replicas disabled
+            Test {
+                policy: DefaultPolicy {
+                    preferred_datacenter: Some("eu".to_owned()),
+                    is_token_aware: true,
+                    permit_dc_failover: true,
+                    fixed_shuffle_seed: Some(123),
+                    ..Default::default()
+                },
+                routing_info: RoutingInfo {
+                    token: Some(Token { value: 160 }),
+                    keyspace: Some(KEYSPACE_NTS_RF_3),
+                    consistency: Consistency::Quorum,
+                    ..Default::default()
+                },
+                // going though the ring, we get order: F , A , C , D , G , B , E
+                //                                      us  eu  eu  us  eu  eu  us
+                //                                      r2  r1  r1  r1  r2  r1  r1
+                expected_groups: ExpectedGroupsBuilder::new()
+                    .deterministic([A, C, G]) // pick + fallback local replicas
+                    .deterministic([F, D, E]) // remote replicas
```

Comment:
How did you find out the output order for the given seed? Purely empirically?

Reply:
I don't assume that the nodes within the group will be in some particular order. I just go over every plan that is being checked and verify that they all have the same ordering of nodes within that group.

Reply:
Oh, right.